esmvaltool/cmor/_fixes/CMIP5/MIROC_ESM.py

```diff
@@ -101,9 +101,6 @@ def fix_metadata(self, cube):
             if time.units.origin == 'days since 0000-01-01 00:00:00':
                 time.units = cf_units.Unit(
                     'days since 1849-01-01 00:00:00', calendar=calendar)
```

Comment:
@jvegasbsc Does this work as expected?

Reply:
We have not been able to test this with actual data. This was ported from version 1 and it looks strange because it is setting  1849 instead of 1850, making me think that maybe there was a different reason for this fix. I will prefer to keep it here until we find that is giving us problems